go.mod

```diff
@@ -105,3 +105,5 @@ require (
 )
 
 retract v0.100.101-test // accidentally published testing version
+
```

Comment:
nit: group retract directives. Any reason we need a newline here?

Reply:
One of the linters / formatters did it, it failed `make presubmit` without it. I can go ahead and make it a retract block though, I imagine that's fine.